app/views/practices/show/authenticated/_show_authenticated.html.erb

```diff
@@ -76,7 +72,6 @@
 
 <div class="practice-mobile-nav padding-1 z-top desktop:display-none display-flex justify-center">
   <%= mail_to main_email, 'Email', cc: cc_emails, id: 'email-link-in-mobile-nav', class: 'width-100 usa-button--outline dm-btn-primary display-flex flex-align-center flex-justify-center height-5 x0-875-right', 'aria-label': "Mobile nav email #{main_email}"  %>
```

Comment:
for the mobile version of the Email button we would also want to remove the `usa-button--outline` class and add the `usa-button` class so it is the primary filled look

Reply:
Another great catch... thx Elisa